### PR TITLE
Upgrade the version of styled-components

### PR DESCRIPTION
**Pull request by sandiiarov**

### CODE REVIEW

File: examples/with-styled-components/pages/_document.js [hunk 1]
```diff
@@ -13,6 +15,10 @@ export default class MyDocument extends Document {
       <html>
         <Head>
           <title>My page</title>
+          <link
```
Comment (timneutkens) [inline examples/with-styled-components/pages/_document.js]: Why use a webfont? 🤔

**Comment by timneutkens** [comment]
styled-components 2.0 is still in alpha/beta right? Should we be upgrading right now? 🤔

**Comment by arunoda** [comment]
@timneutkens yep.
@Alex4S let's do this after they released it.

Basically, we wanted to show about how to use styled-components. I think that idea is there even someone is using the latest beta of styled-components.

**Comment by timneutkens** [comment]
@arunoda in styled-components 2.0 this is fixed https://github.com/zeit/next.js/tree/master/examples/with-styled-components#notes.

**Comment by Alex4S** [comment]
@timneutkens styled-components 2.0 is pre-release now and you are right [this](https://github.com/zeit/next.js/tree/master/examples/with-styled-components#notes) issue is fixed.

**Comment by alexedev** [comment]
Works fine with `2.0.0-1` but has incorrect ids in `2.0.0-4`
screen shot 2017-02-23 at 17 27 45

**Comment by davidrossjones** [comment]
@alexedev Same for me. Issue reported here: https://github.com/styled-components/styled-components/issues/513

**Comment by rovansteen** [comment]
Fyi, the version currently used in the example breaks when using `ThemeProvider` so it might be good idea to update it to the latest `1.x` version, even tho `2.x` will be released soon.

**Comment by timneutkens** [comment]
@rovansteen I'll do that 👍

**Pull request closed without merging**